gitversion/GitVersion.yaml

```diff
@@ -32,7 +31,7 @@ branches:
     increment: None
     prevent-increment-of-merged-branch-version: false
     track-merge-target: false
-    regex: ^((?!master).)*$
+    regex: ^(?!.*master).*$
```

Comment:
This regex is not entirely correct. It filters everything that contains `master`. 

----
Regex: `^(?!.*master).*$`
Inputs:
```
test
master
test_master
test_master_foo
master_test
```
Outputs: 
```
test
```

----

This is the correct regex we need for filtering everything but `master`.
Regex: `^(?!(master)$).*$`
Inputs:
```
test
master
test_master
test_master_foo
master_test
```
Outputs: 
```
test
test_master
test_master_foo
master_test

```

Reply:
thx!